src/main/java/org/humancellatlas/ingest/project/web/ProjectController.java

```diff
@@ -203,4 +199,16 @@
         }
     }
 
+    @GetMapping(path = "projects/filter")
```

Comment:
missing `/` at the beginning (like in all others)

Reply:
I tried adding the `/`, but the endpoint wasn't accessible anymore. Got the `Resource Not Found` error